operate/client/e2e-playwright/tests/processInstanceMigration.spec.ts

```diff
@@ -359,6 +378,7 @@ test.describe.serial('Process Instance Migration', () => {
      */
     await processInstancePage.diagram.clickFlowNode('Task A2');
     await processInstancePage.diagram.showMetaData();
+    await page.waitForTimeout(500); // wait until metadata modal is fully rendered
```

Comment:
Can't you wait for a specific DOM element to appear here?

Timers like this can add unnecessary delay and can be flaky, this should be avoided if possible. Specially in E2E tests

Reply:
I can check. I think the problem was that the metadata modal closed before the monaco editor has finished loading, which was breaking.

Reply:
I'm now waiting for a monaco dom element which seems to be added after monaco was loaded the first time.